chain/chain/src/store.rs

```diff
@@ -1745,6 +1772,27 @@ impl<'a> ChainStoreUpdate<'a> {
             .insert((height, shard_id), chunk_hash);
     }
 
+    pub fn inc_block_refcount(&mut self, block_hash: &CryptoHash) {
+        let refcount = match self.get_block_refcount(block_hash) {
+            Ok(refcount) => *refcount,
+            Err(_) => 0,
```

Comment:
seems not great to swallow the error, we should match only `ErrorKind::DBNotFoundErr` and potentially panic/propagate/at least log the other case (seems to be `IOErr`)

Reply:
++